packages/document/docs/en/blog/release/_meta.json

```diff
@@ -1 +1 @@
-["release-note-1"]
+["release-note-0.1", "release-note-0.3"]
```

Comment:
It is not recommended to include dot in URL, we can replace dot with `_`

Reply:
Fixed